Hercules_DJControl_Inpulse_200.midi.xml

```diff
@@ -0,0 +1,2020 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<MixxxMIDIPreset mixxxVersion="2.2" schemaVersion="1">
+	<info>
+		<name>Hercules DJControl Inpulse 200</name>
+		<author>DJ Phatso for Hercules Technical Support</author>
+		<description>MIDI Preset for Hercules DJControl Inpulse 200</description>
+		<forums>https://www.mixxx.org/forums/viewtopic.php?f=7&amp;t=12592</forums>
+	</info>
+	<controller id="DJControl Inpulse 200">
+		<scriptfiles>
+			<file filename="lodash.mixxx.js"/>
+			<file filename="midi-components-0.0.js"/>
+			<file functionprefix="DJCi200" filename="Hercules-DJControl-Inpulse-200-script.js" />
+		</scriptfiles>
+		<controls>
+
+
+			/****************************************************************************************************/
+			// NN's MIDI Channel 1 (0x90)
+			/****************************************************************************************************/
+
```

Comment:
These aren't comments but XML text nodes. Mixxx' mapping XML parser might currently allow this, but please use actual XML comments here (`<!-- foobar -->`).